### PR TITLE
STR-100 -- BE: Remove someone from the study (remove them from the table)

### PR DESCRIPTION
**Pull request by liadGolan**
# Description
Adds a service function that removes someone from a study
Also rewrote participant tests so that they actually test the functionality
  
# Testing
1. Read over code and make sure everything makes sense
2. Look over all participant tests and ensure that they are testing the functionality they specify

### CODE REVIEW

File: app/Services/ParticipantService.php [hunk 1]
```diff
@@ -34,5 +34,14 @@ public function addGoodParticipantsToParticipantsTable($goodParticipants){
 
 
     }
+
+    public function removeUserFromParticipantsTable($user) {
+        $userParticipant = Participant::where('user_id', $user['user_id'])->first();
+
+        if( $userParticipant != null){
```
Comment (luisjg) [inline app/Services/ParticipantService.php:41]: Shouldn't this also delete the user from the `user_groups` and `participants` table?

**Review by luisjg — COMMENTED** [review]
See comment

File: app/Services/ParticipantService.php (hunk 1 above)
Comment (liadGolan) [reply app/Services/ParticipantService.php:41]: We may want to keep that info later on, once someone no longer has a matching a participant_id, they can no longer log in to the app

**Review by luisjg — APPROVED** [review]
👍

**Review by mbendandour — APPROVED** [review]

**Pull request merged**